.travis.yml

```diff
@@ -2,7 +2,7 @@ language: python
 services:
   - postgresql
 python:
-  - '3.4'
+  - '2.7'
```

Comment:
Is that possible in a travis.yml file? Anyways django-storages only works on 2.7 from what I've read.